src/components/portfolio/DetailsMainPositions.tsx

```diff
@@ -0,0 +1,180 @@
+      height: 'auto',
+      maxHeight: '50rem',
+    },
+    card: {
+      backgroundColor: palette.primary.main,
+      padding: '0 8px',
+    },
+    cardContentUpper: {
+      marginBottom: '2rem',
+    },
+    cardTitle: {
+      color: palette.primary.contrastText,
+      fontSize: '1.2rem',
+      fontWeight: 600,
+    },
+    cardSubtitle: {
+      // TODO: use theme color
+      color: '#EEF1FB',
+      fontSize: '1rem',
+      fontWeight: 500,
+    },
+    cardContentLower: {
+      display: 'flex',
+      justifyContent: 'space-around',
+    },
+    cardAction: {
+      padding: '16px',
+    },
+  })
+);
+
+// type declarations
+type DetailsMainPositionsProps = {
+  positions: Position[];
+};
+
+// returns the details page header
+const DetailsMainPositions: React.FC<DetailsMainPositionsProps> = ({
+  positions,
+}) => {
+  const [posState] = React.useState(positions);
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  // TODO: no hard coded colors
+  // takes a percent value and converts it to a color
+  function convertPercentToColor(val: number): string {
+    return val < 0 ? '#D64745' : '#50E2A8';
+  }
+
+  return (
+    <div>
+      <div className={classes.titleContainer}>
+        <div className={classes.titleWrapper}>
+          <h2 className={classes.sectionTitle}>
+            {t('portfolio.details.positionsTitle')}
+          </h2>
+        </div>
+        <div className={classes.lineWrapper}>
+          <hr className={classes.line} />
+        </div>
+      </div>
+      <div className={classes.gridListWrapper}>
+        <GridList
+          cellHeight="auto"
+          cols={4}
+          spacing={32}
+          className={classes.gridList}
+          style={{ margin: '0 auto' }}
+        >
+          {posState.map((p) => (
+            <GridListTile key={p.stock.isin}>
+              <Card
+                variant="outlined"
+                className={classes.card}
+                style={{ borderColor: convertPercentToColor(p.stock.perf7d) }}
+              >
+                <CardContent>
+                  <div className={classes.cardContentUpper}>
+                    <div className={classes.cardTitle}>{p.stock.name}</div>
+                    <div
+                      className={classes.cardSubtitle}
+                    >{`$${p.stock.price}`}</div>
+                  </div>
+                  <div className={classes.cardContentLower}>
+                    <ValueOverName
+                      value={`${p.stock.perf7d}%`}
+                      name={t('portfolio.details.day7')}
+                      valueColor={convertPercentToColor(p.stock.perf7d)}
+                    />
+                    <ValueOverName
+                      value={`${p.stock.perf1y}%`}
+                      name={t('portfolio.details.year')}
+                      valueColor={convertPercentToColor(p.stock.perf1y)}
+                    />
+                  </div>
+                </CardContent>
```

Comment:
Consider using `react-number-format` to be consistent with the rest of the application.

Reply:
To be done in #105